liquibase-core/src/main/java/liquibase/command/CommandArgumentDefinition.java

```diff
@@ -77,6 +80,14 @@ public boolean isRequired() {
         return required;
     }
 
+    public CommandArgumentDefinition<?> getSupersededBy() {
```

Comment:
@filipelautert helped me understand the reason for the getter/setter `*SupersededBy` methods. I'll put that information here in case others have similar questions.

* The `tag` command requires a URL.
* When executed in the CLI, the user must provide a URL.
* When other _code_ calls `Liquibase.tag()` the caller must provide a `database object`, which in turn must define the URL.
* get/set `*SupersededBy()` allows code to determine if there is a URL defined in either place and throw an error if the `url` parameter is null.